app/models/evaluation.py

```diff
@@ -0,0 +1,97 @@
+from sqlalchemy import String, Text, Integer, DECIMAL, ForeignKey, Date, Enum
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from uuid import UUID
+from .base import Base
+from .user import User
+from .category import CategoryProgress, CategoryNameEnum
+
+
+# RecommendedChallenge 테이블 (추천 챌린지)
+class RecommendedChallenge(Base):
+    __tablename__ = "recommended_challenges"
+    progress_id = Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), ForeignKey("category_progresses.id"), nullable=False
+    )
+    user_id: Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), ForeignKey("CategoryProgresses.user_id"), nullable=False
```

Comment:
아래처럼 `category_progresses`로 통일해야하는 것 같은데 `CategoryProgresses`로 써도 되는건가요?

```suggestion
        UUID(as_uuid=True), ForeignKey("category_progresses.id"), nullable=False
    )
    user_id: Mapped[UUID] = mapped_column(
        UUID(as_uuid=True), ForeignKey("category_progresses.id.user_id"), nullable=False
```

Reply:
저건 이제 해당 클래스의 고유 id 가져올때는 테이블명.id로 가져오고,  다른 키 참조할때는 클래스명.해당키명 을 가져오는거라고 알고있어서 확인 부탁드립니다
@trivialcoding6 @chaeengg

Reply:
![Screenshot_20250405_122351_Samsung Internet](https://github.com/user-attachments/assets/d2031201-b5a1-4882-8796-425bc3ba69a2)
 코드 보면 다른곳도 테이블명으로 되어있는거 같고 참조키 설정에는 테이블 명이 들어가는게 맞는거 같은데 한번 다시 확인 부탁드립니다!

Reply:
제가 잘못 알고있ㄷ건것 같습니다 수정할게욥